testbed/tests/window/test_window.py

```diff
@@ -142,13 +144,92 @@ async def test_move_and_resize(main_window, main_window_probe, capsys):
         finally:
             main_window.content = orig_content
 
-    async def test_full_screen(main_window, main_window_probe):
-        """Window can be made full screen"""
-        main_window.full_screen = True
-        await main_window_probe.wait_for_window("Full screen is a no-op")
+    @pytest.mark.skipif(
+        toga.platform.current_platform == "iOS", reason="Not implemented on iOS"
+    )
```

Comment:
Why is this an explicit platform skipIf, rather than a probe property?

Reply:
Changed it.